tests/python_tests/requirements.txt

```diff
@@ -1,4 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
+transformers<=4.51.3
```

Comment:
I think we should no limit upper bound for transformers on all platforms.
Moreover, it is not convenient to update it regularly in all requirements files.